app/models/lockstep/report_ap_aging_header.rb

```diff
@@ -0,0 +1,7 @@
+class Lockstep::ReportApAgingHeader < Lockstep::ApiRecord
+  self.model_name_uri = "v1/Reports/ap-aging-header"
+  self.id_ref = "group_key"
+  self.query_path=""
```

Comment:
can you pls add space - self.query_path = "", check this across

Reply:
Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/3/commits/bb7622d4d99eb2976f7875442d449eed55f1fcae)